docs/source/conf.py

```diff
@@ -47,6 +47,7 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
     'sphinx_rtd_theme',
+    'myst_parser',
```

Comment:
This may be the root cause of the readthedocs.io build failing. 

It looks like the library is installed as `myst-parser` based on these docs from readthedocs.io: 

https://docs.readthedocs.com/platform/stable/intro/sphinx.html#using-markdown-with-sphinx